app/models/event.rb

```diff
@@ -0,0 +1,8 @@
+class Event < ApplicationRecord
+  belongs_to :creator, class_name: 'User'
+  has_many :attendees, through: :attendance, source: :attendee
+  has_many :attendance, foreign_key: :attended_event_id
```

Comment:
I believe the attendance should be plural because of has_many please make sure of that